app/controllers/api/v1/subscribers_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Api
+  module V1
+    class SubscribersController < ApplicationController
+      def create
+        app = App.find_by(title: app_params)
```

Comment:
сдесь точно нужно работать с id
```suggestion
        app = App.find(params[:id])
```